src/styles/index.css

```diff
@@ -0,0 +1,7 @@
+@import "./reset.css";
+@import "./blocks/card.css";
+@import "./blocks/product_image.css";
+@import "./blocks/product_title.css";
+@import "./blocks/product_rating.css";
+@import "./blocks/product_price.css";
+@import "./blocks/buy_btn.css";
```

Comment:
```suggestion
@import "./blocks/card.css";
@import "./blocks/stars.css";
```

Too complicated structure. Stars need to be a separate block. All other elements belong to the card.